core-bundle/composer.json

```diff
@@ -65,6 +65,7 @@
         "doctrine/orm": "^2.10",
         "doctrine/persistence": "^2.0",
         "dragonmantank/cron-expression": "^2.3",
+        "debril/feed-io": "^6.0",
```

Comment:
You also need to include this change in the monorepo `composer.json`. You can use the `tools/monorepo/vendor/bin/monorepo-tools composer-json` command to automatically make the changes for you.